lib/wallaroo/core/source/gen_source/gen_source.pony

```diff
@@ -122,7 +122,8 @@ actor GenSource[V: Any val] is Source
     layout_initializer: LayoutInitializer,
     metrics_reporter': MetricsReporter iso,
     source_registry: SourceRegistry,
-    disposable_registry: DisposableRegistry)
+    disposable_registry: DisposableRegistry,
+    barrier_coordinator_: BarrierCoordinator)
```

Comment:
Is this used?

Reply:
Removed by commit ea90d7ff97c891fdb5f0c8b06cd238cf49e96b62. I'll mark the other comments related to this arg as resolved, also.